applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_hydraulic_solver.py

```diff
@@ -0,0 +1,692 @@
+            "reform_dofs_at_each_step": false,
+            "consider_periodic_conditions": false,
+            "relative_velocity_tolerance": 1e-3,
+            "absolute_velocity_tolerance": 1e-5,
+            "relative_pressure_tolerance": 1e-3,
+            "absolute_pressure_tolerance": 1e-5,
+            "linear_solver_settings"       : {
+                "solver_type"         : "amgcl"
+            },
+            "volume_model_part_name" : "volume_model_part",
+            "skin_parts": [""],
+            "assign_neighbour_elements_to_conditions": true,
+            "no_skin_parts":[""],
+            "time_stepping"                : {
+                "automatic_time_step" : true,
+                "CFL_number"          : 1,
+                "minimum_delta_time"  : 1e-2,
+                "maximum_delta_time"  : 1.0,
+                "time_step"           : 0.0
+            },
+            "periodic": "periodic",
+            "move_mesh_flag": false,
+            "formulation": {
+                "dynamic_tau": 1.0,
+                "mass_source":true
+            },
+            "artificial_viscosity": false,
+            "artificial_visocosity_settings":{
+                "limiter_coefficient": 1000
+            },
+            "eulerian_fm_ale": true,
+            "eulerian_fm_ale_settings":{
+                "max_CFL" : 1.0,
+                "max_substeps" : 0,
+                "eulerian_error_compensation" : false,
+                "element_type" : "levelset_convection_supg",
+                "element_settings" : {
+                    "dynamic_tau" : 1.0,
+                    "tau_nodal":true
+                }
+            },
+            "levelset_convection_settings": {
+                "max_CFL" : 1.0,
+                "max_substeps" : 0,
+                "eulerian_error_compensation" : false,
+                "element_type" : "levelset_convection_supg",
+                "element_settings" : {
+                    "dynamic_tau" : 1.0,
+                    "tau_nodal":true
+                }
+            },
+            "distance_reinitialization": "variational",
+            "parallel_redistance_max_layers" : 25,
+            "distance_smoothing": false,
+            "distance_smoothing_coefficient": 1.0,
+            "distance_modification_settings": {
+                "model_part_name": "",
+                "distance_threshold": 1e-5,
+                "continuous_distance": true,
+                "check_at_each_time_step": true,
+                "avoid_almost_empty_elements": false,
+                "deactivate_full_negative_elements": false
+            }
+        }""")
+
+        default_settings.AddMissingParameters(super(NavierStokesTwoFluidsHydraulicSolver, cls).GetDefaultParameters())
+        return default_settings
+
+    def __init__(self, model, custom_settings):
+        # TODO: DO SOMETHING IN HERE TO REMOVE THE "time_order" FROM THE DEFAULT SETTINGS BUT KEEPING THE BACKWARDS COMPATIBILITY
```

Comment:
Not sure if it applies but you don't need to keep backwards compatibility anymore.